### ISSUE TITLE
Not working - NoMethodError

### ISSUE THREAD

**Issue by syedhassan** [BaseSecrete/rorvswild#13]
Hi,

When I start the access the app then I get the following error: FYI, I can only use the gem version **1.5.1**

```
NoMethodError - undefined method `[]' for nil:NilClass:
  rorvswild (1.5.3) lib/rorvswild/plugin/action_controller.rb:17:in `start'
  activesupport (4.2.0) lib/active_support/notifications/fanout.rb:98:in `start'
  activesupport (4.2.0) lib/active_support/notifications/fanout.rb:42:in `block in start'
  activesupport (4.2.0) lib/active_support/notifications/fanout.rb:42:in `each'
  activesupport (4.2.0) lib/active_support/notifications/fanout.rb:42:in `start'
  activesupport (4.2.0) lib/active_support/notifications/instrumenter.rb:31:in `start'
  activesupport (4.2.0) lib/active_support/notifications/instrumenter.rb:18:in `instrument'
  activesupport (4.2.0) lib/active_support/notifications.rb:164:in `instrument'
  actionpack (4.2.0) lib/action_controller/metal/instrumentation.rb:30:in `process_action'
  actionpack (4.2.0) lib/action_controller/metal/params_wrapper.rb:250:in `process_action'
  activerecord (4.2.0) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
  actionpack (4.2.0) lib/abstract_controller/base.rb:137:in `process'
  actionview (4.2.0) lib/action_view/rendering.rb:30:in `process'
  actionpack (4.2.0) lib/action_controller/metal.rb:195:in `dispatch'
  actionpack (4.2.0) lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
  actionpack (4.2.0) lib/action_controller/metal.rb:236:in `block in action'
  actionpack (4.2.0) lib/action_dispatch/routing/route_set.rb:73:in `dispatch'
  actionpack (4.2.0) lib/action_dispatch/routing/route_set.rb:42:in `serve'
  actionpack (4.2.0) lib/action_dispatch/journey/router.rb:43:in `block in serve'
  actionpack (4.2.0) lib/action_dispatch/journey/router.rb:30:in `each'
  actionpack (4.2.0) lib/action_dispatch/journey/router.rb:30:in `serve'
  actionpack (4.2.0) lib/action_dispatch/routing/route_set.rb:802:in `call'
```

FYI, I am using Ruby 2.3.1,  Rails 4.2

**Comment by alexisbernard** [comment]
It's seems to be the same issue as #12. The version below should work (similar to 1.5.1). Can you confirm please ?

```ruby
gem "rorvswild", git: "https://github.com/BaseSecrete/rorvswild.git", ref: "db5187bf7da991d80afc3f6aef1fb9fac7baaa91
```

**Comment by syedhassan** [comment]
ok, will the new versions be made backwards compatible?

**Comment by alexisbernard** [comment]
I have released version 1.5.4.
